media-sound/subsonic-bin/subsonic-bin-6.1.6.ebuild

```diff
@@ -1,35 +1,31 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-
-inherit eutils user
+EAPI=7
 
 MY_PN="${PN//-bin}"
 
 DESCRIPTION="Subsonic is a complete, personal media streaming solution"
 HOMEPAGE="http://www.subsonic.org/"
 SRC_URI="mirror://sourceforge/${MY_PN}/${PV}/${MY_PN}-${PV}-standalone.tar.gz"
 
-LICENSE="GPL-3"
+LICENSE="all-rights-reserved"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="ffmpeg lame selinux"
+IUSE="+ffmpeg +lame selinux"
 
 DEPEND=""
-RDEPEND="virtual/jre
+RDEPEND="
+	acct-group/subsonic
+	acct-user/subsonic
+	virtual/jre
 	lame? ( media-sound/lame )
 	ffmpeg? ( media-video/ffmpeg )
-	selinux? ( sec-policy/selinux-subsonic )"
+	selinux? ( sec-policy/selinux-subsonic )
+"
 
 S="${WORKDIR}/"
```

Comment:
And while we are at it, there is no need for the trailing `/`

Reply:
Because in large lists it looks clear. The first line in the list is skipped so that the indentation lines up the list in one column, and the trailing quotation mark is never forgotten when re-sorting the list or adding a new element to the end. Portage uses both styles.
![Screenshot_9](https://user-images.githubusercontent.com/2046529/94205819-c0398500-fecc-11ea-95e6-a0d8c9d7063e.png)

Reply:
`S="${WORKDIR}"`

Reply:
oh, fixed